src/main/webapp/i18n/de/exam.json

```
@@ -312,9 +312,12 @@
             "pendingChanges": "Alle ungespeicherten Änderungen gehen beim Verlassen der Klausur verloren. Bist du sicher, dass du fortfahren möchtest?",
             "handInEarly": "Vorzeitig abgeben",
             "continueAfterHandInEarly": "Weitermachen",
+            "attendanceCheck": "Anwesenheitskontrolle",
             "continueAfterHandInEarlyDescription": "Du willst doch nicht vorzeitig abgeben?",
             "handInEarlyNoticeFirstSentence": "Du bist dabei die Klausur vorzeitig abzugeben.",
             "handInEarlyNoticeSecondSentence": "Du kannst danach nicht mehr an der Klausur teilnehmen.",
+            "attendanceNotVerifiedWarningFirstSentence": "Deine Anwesenheit wurde noch nicht kontrolliert.",
+            "attendanceNotVerifiedWarningSecondSentence": "Bitte kontaktiere einen/eine Tutor:in nach der Prüfung, um deine Anwesenheit zu kontrollieren.",
```

```suggestion
            "attendanceNotVerifiedWarningSecondSentence": "Bitte kontaktiere die Hörsaalaufsicht nach der Prüfung, um deine Anwesenheit zu kontrollieren.",
```
Or at least keep it in line with the English version (instructor!=Tutor:in)